### ISSUE TITLE
Slider adjustment is unresponsive

### ISSUE THREAD

**Issue by jarnoh** [jarnoh/knobroom#3]
Adjusting Lightroom sliders is slow and unresponsive, Lightroom provides only few image updates per second.

**Comment by nicksnips** [comment]
I agree-- using the sliders manually gives great feedback, but via the plugin it's quite slow; basically not useable.

**Comment by davyboyhayes** [comment]
I find that after LR updated to use the Graphics Card to offload a bunch of on the fly processing, this is a lot less of an issue.
